content/20.pharmaceuticals.md

```diff
@@ -243,7 +244,23 @@ Patients who received HCQ also had a longer duration of hospitalization than pat
 As a result, enrollment in the HCQ arm of the RECOVERY trial was terminated early [@url:https://www.recoverytrial.net/news/statement-from-the-chief-investigators-of-the-randomised-evaluation-of-covid-19-therapy-recovery-trial-on-hydroxychloroquine-5-june-2020-no-clinical-benefit-from-use-of-hydroxychloroquine-in-hospitalised-patients-with-covid-19].
 The story of CQ/HCQ therefore illustrates how initial promising _in vitro_ analyses can fail to translate to clinical usefulness.
 
-While CQ/HCQ are well-known medications that have long been prescribed in certain contexts, investigation of another well-established type of pharmaceutical was facilitated by the fact that it was already being taken by a large number of COVID-19 patients.
+A similar story has arisen with the broad-spectrum, small-molecule anthelmintic ivermectin, which is a synthetic analog of avermectin, a bioactive compound produced by a microorganism known as _Streptomyces avermectinius_ and _Streptomyces avermitilis_ [@doi:10.1038/nrmicro1048; @doi:10.1128/AAC.15.3.361].
+Avermectin disrupts the ability of parasites to avoid the host immune response by blocking gutamate-gated chloride ion channels in the peripheral nervous system from closing, leading to hyperpolarization of neuronal membranes, disruption of neural transmission, and paralysis [@doi:10.2183/pjab.87.13; @doi:10.1038/nrmicro1048; @doi:10.1038/ja.2017.11].
+Ivermectin has been used since the early 1980s to treat endo- and ecto-parasitic infections by helminths, insects, and arachnids in veterinary contexts [@doi:10.1038/nrmicro1048; @doi:10.1128/AAC.15.3.372] and since the late 1980s to treat human parasitic infections as well [@doi:10.1038/nrmicro1048; @doi:10.2183/pjab.87.13].
+More recent research has indicated that ivermectin might function as a broad-spectrum antiviral by distrupting the trafficking of viral proteins by both RNA and DNA viruses [@doi:10.1038/s41429-020-0336-z; @doi:10.1038/ja.2017.11; @doi:10.1186/s12941-020-00368-w], although most of these studies have demonstrated this effect _in vitro_ [@doi:10.1186/s12941-020-00368-w].
+The potential for antiviral effects on SARS-CoV-2 were investigated _in vitro_, and ivermectin was found to inhibit viral replication in a cell line derived from Vero cells (Vero-hSLAM) [@doi:10.1016/j.antiviral.2020.104787] but at a concentration that is much higher than that explored by existing dosage guidelines [@doi:10.4269/ajtmh.20-0271; @doi:10.1002/cpt.1889] and likely to be associated with significant side effects due to the increased potential that the compound could cross the mammalian blood-brain barrier [@doi:10.1080/13102818.2020.1775118; @doi:10.1371/journal.pntd.0001883].
+
+Retrospective studies and small RCTs began investigating the effects of standard doses of this low-cost, widely available drug.
```

Comment:
We may not introduce the RCT acronym until later (maybe the summary table)?

Reply:
Good catch! Now addressed in #1001